go.mod

```diff
@@ -5,10 +5,10 @@ go 1.13
 require (
 	github.com/golang/mock v1.3.1
 	github.com/magefile/mage v1.9.0
-	github.com/mattermost/mattermost-server/v5 v5.3.2-0.20200627210001-eff2209a7e37
+	github.com/mattermost/mattermost-server/v5 v5.3.2-0.20200714073806-915a31cd8a65
```

Comment:
This module version is temporal for the tests to run correctly. It will change after the server PR gets merged